docs/src/man/basics.md

```diff
@@ -0,0 +1,1220 @@
+# First Steps with DataFrames
+
+The first and imporatant part to work with DataFrames.jl, is to install the `DataFrames.jl`
+package which is available thorugh the Julia package system and can be installed using the
+following commands:
+
+```julia
+julia> using Pkg
+julia> Pkg.add("DataFrames")
+```
+
+Or,
+
+```julia
+julia> ]
+
+julia> add DataFrames
+```
+
+To make sure everything works as expected, try to load the package and if you have the time execute
+its test suits:
+```julia
+julia> using DataFrames
+julia> using Pkg
+julia> Pkg.test("DataFrames")
+```
```

Comment:
additionally the user **must** check the version installed with `st` command and ensure it is `1` release.